src/components/Header.jsx

```diff
@@ -108,17 +111,25 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const links = {
-  Scholarships: '/scholarships',
-  Add: '/scholarships/new',
-  About: '/about',
-  Contact: '/contact',
+  'navbar.scholarships': '/scholarships',
+  'navbar.add': '/scholarships/new',
+  'navbar.about': '/about',
+  'navbar.contact': '/contact',
+};
+
+const languages = {
+  English: 'en',
+  Español: 'es',
```

Comment:
Nit: Not sure how well special characters like ñ are supported as keys (seems to work here) but perhaps we can flip this so that it maps from language code to the name of the language? If we ever decide to support other languages with more complex characters like 한국어 or 中文 I'm not sure how well they'll work as keys (maybe they can as strings).